crawl-ref/source/dat/descript/species.txt

```diff
@@ -91,9 +91,9 @@ Kobold
 
 Kobolds are small, mischievous and a little fragile, but multi-talented.
 %%%%
-Mountain Dwarf
+Sand Dwarf
```

Comment:
The rename has caused this to not be in alphabetical order. Put this entry after 'Ogre' instead.

Reply:
done